spec/u-namespace.md

```diff
@@ -0,0 +1,86 @@
+# MessageFormat 2.0 Unicode Namespace
+
+The `u:` _namespace_ is reserved for the definition of _options_
+which affect the _function context_ of the specific _expressions_
+in which they appear,
+or for the definition of _options_ that are universally applicable
+rather than function-specific.
```

Comment:
We also reserve `u:` functions (even if there are none currently).

Reply:
@aphillips Do you think we ought to call that out specifically?

Reply:
Yeah, I think we should mention it. Perhaps add a sentence to the end like:

> It might also be used to define _functions_ in a future release.

Reply:
```suggestion
rather than function-specific.
It might also be used to define _functions_ in a future release.
```